mnefun/_mnefun.py

```diff
@@ -2429,246 +2426,6 @@ def _viz_raw_ssp_events(p, subj, ridx):
-        struc = structurals[si]
-        report = Report(verbose=False)
-        print('  Processing subject %s/%s (%s)'
-              % (si + 1, len(subjects), subj))
-
-        # raw
-        fnames = get_raw_fnames(p, subj, 'raw', erm=False, add_splits=True,
-                                run_indices=run_indices[si])
-        if not all(op.isfile(fname) for fname in fnames):
-            raise RuntimeError('Cannot create reports until raw data exist')
-        raw = mne.concatenate_raws(
-            [read_raw_fif(fname, allow_maxshield='yes')
-             for fname in fnames])
-
-        # sss
-        sss_fnames = get_raw_fnames(p, subj, 'sss', False, False,
-                                    run_indices[si])
-        has_sss = all(op.isfile(fname) for fname in sss_fnames)
-        sss_info = mne.io.read_info(sss_fnames[0]) if has_sss else None
-
-        # pca
-        pca_fnames = get_raw_fnames(p, subj, 'pca', False, False,
-                                    run_indices[si])
-        has_pca = all(op.isfile(fname) for fname in pca_fnames)
-
-        with plt.style.context(style):
-            ljust = 25
-            #
-            # Head coils
-            #
-            section = 'HPI coil SNR'
-            if p.report_params.get('coil_snr', True):
-                t0 = time.time()
-                print(('    %s ... ' % section).ljust(ljust), end='')
-                if p.report_params['coil_t_step'] == 'auto':
-                    t_step = raw.times[-1] / 100.  # 100 points
-                else:
-                    t_step = float(p.report_params['coil_t_step'])
-                fig = plot_good_coils(raw, t_step, show=False)
-                fig.set_size_inches(10, 2)
-                fig.tight_layout()
-                report.add_figs_to_section(fig, section, section,
-                                           image_format='svg')
-                print('%5.1f sec' % ((time.time() - t0),))
-            else:
-                print('    %s skipped' % section)
-
-            #
-            # Head movement
-            #
-            section = 'Head movement'
-            if p.report_params.get('head_movement', True):
-                print(('    %s ... ' % section).ljust(ljust), end='')
-                t0 = time.time()
-                trans_to = _load_trans_to(p, subj, run_indices[si], raw)
-                pos = [_headpos(p, fname) for ri, fname in enumerate(fnames)]
-                fig = plot_head_positions(pos=pos, destination=trans_to,
-                                          info=raw.info, show=False)
-                del trans_to
-                fig.set_size_inches(10, 6)
-                fig.tight_layout()
-                report.add_figs_to_section(fig, section, section,
-                                           image_format='svg')
-                print('%5.1f sec' % ((time.time() - t0),))
-            else:
-                print('    %s skipped' % section)
-
-            #
-            # PSD
```

Comment:
Ok last feature request I swear. We should probably check for `bads` list and add them for `PSD` esp considering our dead sensor `MEG1743`

![image](https://user-images.githubusercontent.com/4381290/34062501-bd79acc8-e1a1-11e7-8143-63418f5556d8.png)

Reply:
Added, feel free to try again